lib/pages/login/login_page.dart

```diff
@@ -2,10 +2,11 @@ import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:flutter_survey/gen/assets.gen.dart';
 import 'package:flutter_survey/gen/colors.gen.dart';
-import 'package:flutter_survey/resouces/dimens.dart';
+import 'package:flutter_survey/resources/dimens.dart';
+import 'package:flutter_survey/routes.dart';
 
-import 'widgets/blur_background.dart';
-import 'widgets/dimmed_background.dart';
+import '../widgets/blur_background.dart';
+import '../widgets/dimmed_background.dart';
```

Comment:
maybe having a dedicated `background/`? 💁‍♂️

Reply:
Same, I prefer to keep them here until we got more widgets that need to be moved. Noted 🥲